src/node/install.sh

```diff
@@ -232,6 +232,8 @@ else
         [ ! -z "$https_proxy" ] && npm set https-proxy="$https_proxy"
         [ ! -z "$no_proxy" ] && npm set noproxy="$no_proxy"
         npm install -g pnpm
+        pnpm setup
```

Comment:
oops, my motivation is that when I open a project in a nodejs devcontainer, from reading <https://pnpm.io/npmrc#node-modules-settings> I expect the pnpm store-dir to be set in the following order:

If the $PNPM_HOME env variable is set, then $PNPM_HOME/store
If the $XDG_DATA_HOME env variable is set, then $XDG_DATA_HOME/pnpm/store
On Windows: ~/AppData/Local/pnpm/store
On macOS: ~/Library/pnpm/store
On Linux: ~/.local/share/pnpm/store

What actually happens is that since $PNPM_HOME is not set yet (happens during `pnpm setup` from my understanding), that the pnpm store-dir is set to the project root

Checkout this issue: <https://github.com/pnpm/pnpm/issues/7100>

Reply:
Thanks for the explanation.

Can you add few tests to validate your changes?